tools/parse/src/main/scala/org/allenai/nlpstack/parse/poly/eval/ParseAnalyzer.scala

```diff
@@ -0,0 +1,116 @@
+package org.allenai.nlpstack.parse.poly.eval
+
+import org.allenai.nlpstack.parse.poly.polyparser.PolytreeParse
+
+/** A ParseAnalyzer maps a candidate parse to an "analysis", i.e. a histogram.
+  *
+  * For instance, this histogram may be the count of mistaken arclabels or part-of-speech tags.
+  */
+trait ParseAnalyzer extends (PolytreeParse => Map[String, Double]) {
```

Comment:
Looks like all the analyzers below are populating the histogram with integer values. The histogram values are not normalized in any way. Do we need a `Double` ?